.devcontainer/Dockerfile

```diff
@@ -0,0 +1,29 @@
+FROM ubuntu:24.04
+
+RUN apt update -y && apt install -y \
+	gcc \

```

Comment:
if you want to reduce the size of the image down to about 656mb, you can swap out `apt install -y` with `apt install -y --no-install-recommends` and add `ca-certificates` to the list of packages to install (as this is needed to clone the git repositories later).

Reply:
That sounds good to me!